next.config.js

```diff
@@ -0,0 +1,11 @@
+const withMdxEnhanced = require('next-mdx-enhanced')
+
+module.exports = withMdxEnhanced({
+  extendFrontMatter: {
+    process: (_, frontMatter) => ({
+      id: frontMatter.__resourcePath.replace('.mdx', '')
+    })
+  }
+})({
+  pageExtensions: ['ts', 'tsx', 'js', 'jsx', 'md', 'mdx']
```

Comment:
Silly question but how does know to ignore `.test.tsx` files?

Reply:
It doesn't, I'll look into this when I get the `mdx` from node_modules.